### PR TITLE
[16.0][MIG] partner_disable_gravatar

### PR DESCRIPTION
**Pull request by legalsylvain**
Trivial migration of ``partner_disable_gravatar`` that disable call of gravatar.com when entering email on partner form view.

- bump version in manifest
- update translation
- remove useless CONFIGURE.rst file

CC : @alexis-via

Thanks for your review.

### CODE REVIEW

File: partner_disable_gravatar/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Partner Disable Gravatar",
-    "version": "14.0.1.0.0",
+    "version": "16.0.1.0.1",
```
Comment (gdgellatly) [inline partner_disable_gravatar/__manifest__.py:7]: why .1?

File: partner_disable_gravatar/i18n/partner_disable_gravatar.pot [hunk 2]
```diff
@@ -16,19 +18,4 @@ msgstr ""
...
-#. module: partner_disable_gravatar
-#: model:ir.model.fields,field_description:partner_disable_gravatar.field_res_partner__id
-msgid "ID"
-msgstr ""
-
-#. module: partner_disable_gravatar
-#: model:ir.model.fields,field_description:partner_disable_gravatar.field_res_partner____last_update
-msgid "Last Modified on"
-msgstr ""
+msgstr ""
```
Comment (gdgellatly) [inline partner_disable_gravatar/i18n/partner_disable_gravatar.pot:21]: Is it correct no newline?

**Review by alexis-via — APPROVED** [review]

File: partner_disable_gravatar/i18n/partner_disable_gravatar.pot (hunk 2 above)
Comment (legalsylvain) [reply partner_disable_gravatar/i18n/partner_disable_gravatar.pot:21]: pre-commit check say yes.

File: partner_disable_gravatar/__manifest__.py (hunk 1 above)
Comment (legalsylvain) [reply partner_disable_gravatar/__manifest__.py:7]: for some obscur pypi reasons. (afaik).

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Comment by alexis-via** [comment]
@OCA-git-bot hey stupid bot, why do you consider that i'm not allowed to merge ? I'm the maintainer of that module, no ?

**Pull request merged**